client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -29,6 +30,13 @@ export default function Dashboard(): JSX.Element {
 
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
+      <Navbar
```

Comment:
Please use enums to define routes, see article below for reference.

https://betterprogramming.pub/the-best-way-to-manage-routes-in-a-react-project-with-typescript-c4e8d4422d64

Reply:
fixed the naming as per the standard
created 5 routes and 5 strings which can be modified and re-used